setup/teardown.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Teardown
+description: Learn how to teardown Coder and its associated infrastructure.
+---
+
+This guide shows you how to teardown Coder and the cluster it is deployed on.
+
+**Note:** these instructions help you remove infrastructure spun up via our
+[Kubernetes Documentation](https://coder.com/docs/setup/kubernetes), and do not
+include teardown steps for any additional cluster resources that may have been deployed.
```

Comment:
```suggestion
include teardown steps for any additional resources that may have been created
outside of these steps
```

Just a suggestion, but it's not necessarily cluster resources. Things like IAM settings and other network security items.

Reply:
Makes sense - I'll include.